src/game_stats.py

```diff
@@ -148,6 +148,7 @@ def _create_msg(self):
             totals["chickens"] += stats["chickens"]
             totals["deaths"] += stats["deaths"]
             totals["merc_deaths"] += stats["merc_deaths"]
+            totals["failed_runs"] += stats["failed_runs"]
```

Comment:
Hmm, that was an easy solution to keep the number of failed runs to a minimum... oh well.